extensions/blocks/likes/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Internal dependencies
+ */
+import JetpackLikesAndSharingPanel from '../../shared/jetpack-likes-and-sharing-panel';
+import LikesCheckbox from './likes-checkbox';
+
+export const name = 'likes';
+
+export const settings = {
+	render: () => {
```

Comment:
Observation (this is fine as is — no need to change): If an arrow function returns a single expression, we can use the short form:

```js
	render: () => (
		<JetpackLikesAndSharingPanel>
			<LikesCheckbox />
		</JetpackLikesAndSharingPanel>
	),
```

Reply:
Since this function is an object property, and isn't referencing `this` in the function body, we might alternatively shorten to this (method-like) notation:

```suggestion
	render() {
```